libbpf-tools/biolatency.bpf.c

```diff
@@ -52,17 +55,30 @@ int trace_rq_start(struct request *rq)
 }
 
 SEC("tp_btf/block_rq_insert")
-int BPF_PROG(block_rq_insert, struct request_queue *q, struct request *rq)
+int BPF_PROG(block_rq_insert)
 {
-	return trace_rq_start(rq);
+	if (LINUX_KERNEL_VERSION > KERNEL_VERSION(5, 10, 0))
+		return trace_rq_start((void *)ctx[0]);
+	else
+		return trace_rq_start((void *)ctx[1]);
 }
 
 SEC("tp_btf/block_rq_issue")
-int BPF_PROG(block_rq_issue, struct request_queue *q, struct request *rq)
+int BPF_PROG(block_rq_issue)
 {
-	if (targ_queued && BPF_CORE_READ(q, elevator))
-		return 0;
-	return trace_rq_start(rq);
+	struct request *rq;
+
+	if (LINUX_KERNEL_VERSION > KERNEL_VERSION(5, 10, 0)) {
```

Comment:
please drop biolatency.bpf.c changes, they are being addressed in https://github.com/iovisor/bcc/pull/3317, no need to do it in two places